pkg/api/handlers/compat/containers_archive.go

```diff
@@ -92,11 +94,13 @@ func handleHeadAndGet(w http.ResponseWriter, r *http.Request, decoder *schema.De
 
 func handlePut(w http.ResponseWriter, r *http.Request, decoder *schema.Decoder, runtime *libpod.Runtime) {
 	query := struct {
-		Path string `schema:"path"`
+		Path       string `schema:"path"`
+		CopyUIDGID bool   `schema:"copyUIDGID"`
```

Comment:
Can we name it `chown`?

Otherwise, `chown`, `archive` and `copyUIDGID` all refer to the same thing.

Reply:
We could.